### ISSUE TITLE
A security borg can put handcuffs on a person that is missing a hand.

### ISSUE THREAD

**Issue by Alriac** [ParadiseSS13/Paradise#10644]
**Problem Description**:
Characters with only one hand can't be handcuffed with handcuffs or cable restrains. But a security borg will be able to put restrains on someone that is missing a hand.

What is the problem?
A character that is missing one hand can be handcuffed by a security borg.

**What did you expect to happen**: 
The borg is unable to put handcuffs on my character.

Why do you think this is an issue?
Its incoherent with regular handcuffs and also makes no sense being able to put handcuffs on someone that is missing a hand.

**What happened instead**:
A borg was able to put zipties on my character with no issue.

Why do you think this is an important issue?
Its not physically possible to put handcuffs on someone that is missing a hand. Its just wrong,

**Steps to reproduce the problem**:
1- Enter the game with a character that is missing a hand (mine was missing a left hand)
2- Find a security borg.
3- Commit a crime in front of it.
4- Let it put restrains on you.
5- You will be restrained like a regular two-handed greytider.

**When did the problem start happening**:
First time I saw it was 12/01/2019. I dont know for how long it has been happening since its the first time I was in this situation.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was anything else out of the ordinary happening?)**:
My character was handcuffed and put in processing, where an officer took all the items from it including the clothes. After a while, the officer put all the items and clothes on my character again, and he said that my characted was invisible but it was perfectly visible for me and nobody else seem to notice it.

**Comment by lass9030** [comment]
Working As Intended™️

**Comment by Shadeykins** [comment]
They cuff you to your belt-loops or something. /shrug

This is more a balance implication than anything, because it would let people choose to have one arm and be completely uncuffable as a result.

**Comment by Alriac** [comment]
If that is working as intended maybe the issue is that humans can't put restrains on someone who is missing a single hand?
I still find incoherent that humans can only put restrains on someone who have both hands while borgs can do it on someone missing one hand. I think one way or another it should be made equal.

I located the commit where this was added, allowing both borgs and humans to put restrains on anyone who is missing a single hand: https://github.com/ParadiseSS13/Paradise/commit/f7b298f68e88880b1fb7dc78f6b0c29db9975bb5#diff-59a07870f8bd3a0ebe35d335d49994c6

And the commit where that changed for humans but not for borgs: https://github.com/ParadiseSS13/Paradise/commit/8c5beaaaff8bd581358332216ec332c4c8e806b3#diff-59a07870f8bd3a0ebe35d335d49994c6

It seems like a mistake to me, when replacing H.get_organ("l_hand") with H.has_left_hand() for humans but not for borgs, as the commit comment doesn't mention anything about balancing, just the new procs.
